src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -2143,6 +2143,35 @@ class Settings extends Component {
                   onClick={this.handleSubmit}
                 />
               )}
+              {this.state.selectedSetting !== 'Account' ? (
+                ''
+              ) : (
+                <div style={{ marginRight: '20px' }}>
+                  {UserPreferencesStore.getTheme() === 'light' ? (
+                    <hr
+                      className="break-line-light"
+                      style={{ height: '2px', marginTop: '25px' }}
+                    />
+                  ) : (
+                    <hr className="break-line-dark" />
+                  )}
+
+                  <p
+                    style={{
+                      textAlign: 'center',
+                      marginTop: '20px',
+                      display: 'flex',
+                      justifyContent: 'center',
```

Comment:
No need to centre it, align it like rest of the fields

Reply:
@fragm3 The style is followed the same way as on accounts page. We should follow similar alignment